<launch>

	<node ns="arduino_zero" name="nero_arduino_zero" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM0" />
    <node ns="arduino_one" name="nero_arduino_zero" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM1" />

	<node pkg="i_robot" type="main_executable" name="main_node" output="screen"/>  
    <node pkg="i_robot" type="controller_executable" name="joy_controller_node" output="screen" />  

	<node pkg="joy" type="joy_node" name="joy_receive_node" output="screen">

       <rosparam>
          dev: /dev/input/js2
          deadzone: 0.2
       </rosparam>   
		
    </node>


</launch>








